server_test.go

```diff
@@ -180,7 +217,7 @@ func TestServer(t *testing.T) {
 	}
 
 	if len(be.messages) != 1 {
-		t.Fatal("Invalid number of sent messages:", be.messages)
+		t.Fatal("Invalid number of sent messages:", be.messages, be.anonmsgs)
```

Comment:
Please check that the number of `anonmsgs` is zero too.

Reply:
Changed in e198df1